mc-sema/cfgToLLVM/x86Instrs_MOV.cpp

```diff
@@ -564,6 +564,18 @@ static InstTransResult translate_MOV64ri(NativeModulePtr natM, BasicBlock *& blo
     return ret ;
 }
 
+template <int width>
+int GET_XAX() {
+  if (64 == width) {
+    return X86::RAX;
+  } else if (32 == width) {
+    return X86::EAX;
+  } else if (16 == width) {
+    return X86::AX;
+  } else {
+    return X86::AL;
```

Comment:
Based on the uses in the dispatch map, it's not possible for it to receive `128`.